src/main/kotlin/no/nav/familie/ef/sak/journalføring/JournalpostUtil.kt

```diff
@@ -0,0 +1,39 @@
+package no.nav.familie.ef.sak.journalføring
+
+import no.nav.familie.kontrakter.ef.sak.DokumentBrevkode
+import no.nav.familie.kontrakter.felles.Behandlingstema
+import no.nav.familie.kontrakter.felles.journalpost.DokumentInfo
+import no.nav.familie.kontrakter.felles.journalpost.Dokumentvariantformat
+import no.nav.familie.kontrakter.felles.journalpost.Journalpost
+
+fun Journalpost.harStrukturertSøknad(): Boolean {
```

Comment:
Burde denne kanskje få en test eller to? :P

Reply:
Ja, det burde den. Indirekte testet, men det er litt tynt